dask/compatibility.py

```diff
@@ -253,3 +253,11 @@ class to receive bound method
         setattr(cls, name, types.MethodType(func, None, cls))
     else:
         setattr(cls, name, func)
+
+
+try:
+    from importlib import import_module
+except ImportError:
+    def import_module(name):
+        __import__(name)
+        return sys.modules[name]
```

Comment:
Is this necessary to add to compatibility.py?  IIRC, we did this in `toolz` to support Python 2.6.  I believe Python 2.7 and 3 have `importlib.import_module`.